### PR TITLE
docs: add guide for using supabase cli to inspect any Postgres db

### PR DESCRIPTION
**Pull request by loong**
ToDos:

 - [x] Add links to [reference pages](https://github.com/supabase/cli/pull/1348) once merged and references rebuild

### CODE REVIEW

File: apps/docs/pages/guides/database/inspect.mdx
Comment (KevinBrolly) [inline apps/docs/pages/guides/database/inspect.mdx:152]: Should be "shows statements" rather than "shows queries"

File: apps/docs/pages/guides/database/inspect.mdx
Comment (KevinBrolly) [inline apps/docs/pages/guides/database/inspect.mdx:153]: Should be "shows statements" rather than "shows queries"

**Review by KevinBrolly — COMMENTED** [review]
Looking great, 2 tiny comments about language, ready to merge once the links to reference docs for all commands are in place 👍

File: apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts
Comment (kiwicopple) [inline apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts:553]: perhaps there is a clearer title for this one -  he goal in the title is to give the reader a "hook", so that they know why they would read it. 

Perhaps something like "Monitoring performance, indexes, and Postgres internals"? (or something similar)

File: apps/docs/pages/guides/database/inspect.mdx
Comment (kiwicopple) [inline apps/docs/pages/guides/database/inspect.mdx:10]: it's good to start with a paragraph rather than a title - I think we can remove this one and it will give a better flow from the main page title

File: apps/docs/pages/guides/database/inspect.mdx
Comment (kiwicopple) [inline apps/docs/pages/guides/database/inspect.mdx:14]: Let's remove this section and then just link to the installation steps in the CLI Guides

**Review by kiwicopple — APPROVED** [review]
epic PR @loong - a few comments, but otherwise this is a huge addition for LW

File: spec/cli_v1_commands.yaml
Comment (loong) [inline spec/cli_v1_commands.yaml:4]: @sweatybridge is this okay? Or should we wait for the new release and then generate the reference docs with the right semver

Comment (sweatybridge) [reply spec/cli_v1_commands.yaml:4]: We can just keep the current semver since only docs are updated.

**Pull request merged**
